### PR TITLE
Adding recipe for iotaa

### PR DESCRIPTION
**Pull request by maddenp**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged. **NA**
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by maddenp** [comment]
I am willing to be a maintainer.

**Comment by maddenp** [comment]
@conda-forge-admin, please ping team

**Comment by maddenp** [comment]
Not sure if I'm phrasing this right, but:

@conda-forge-admin, please ping team conda-forge/help-python

File: recipes/iotaa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% set version = "0.7.2" %}
+{% set build = "0" %}
+{% set python = "python >=3.8" %}
```
Comment (ocefpaf) [inline recipes/iotaa/meta.yaml:3]: Please don't template these. They obfuscate the recipe and don't help much.

**Review by ocefpaf — COMMENTED** [review]
The error is b/c you have a conda recipe in the tarball and conda-build is trying to build it too. You should delete it.

File: recipes/iotaa/meta.yaml [hunk 2]
```diff
@@ -1,28 +1,24 @@
-{% set version = "0.7.2" %}
-{% set build = "0" %}
-{% set python = "python >=3.8" %}
-
 package:
   name: iotaa
-  version: {{ version }}
```
Comment (ocefpaf) [inline recipes/iotaa/meta.yaml]: The version should be in the jinja template! My comment was pointing to the python version and build number only. Sorry for the confusion but when in doubt, check our docs and examples.

Comment (maddenp) [reply recipes/iotaa/meta.yaml]: Understood, will fix, thanks.

The Linux/OSX builds fail trying to run the `.exe`, so I will revert to the separate tests with selectors, unless you have another idea.

I tried adding an `rm -rf recipe`, which is fine on Linux/OSX, but which of course does not work on Windows, which I am also no expert on. Would it be bad practice to omit a Windows build for this package? If so I will find a solution.

Comment (ocefpaf) [reply recipes/iotaa/meta.yaml]: The idea is to not use the `.exe` and see if Windows can call the executable without it. As far as I know it is possible to run, on Windows, with both `cli` or `cli.exe`. The former will save you an extra line in the recipe and the need for the OS selector.

File: recipes/iotaa/meta.yaml (hunk 1 above)
Comment (maddenp-noaa) [reply recipes/iotaa/meta.yaml]: Fixed.

**Pull request merged**